coop_membership/models/res_company.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2016-Today: La Louve (<http://www.lalouve.fr/>)
+# @author: La Louve
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html
+
+from odoo import api, fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    contact_us_message = fields.Html(
+        string="Contact Us Message",
+        translate=True,
+        default=lambda self: self.get_default_message()
```

Comment:
```suggestion
        default=(
            'Hello,<br/>'
            'Please contact an employee or go at the members\' office '
            'for administrative reasons.<br/>' 
            'Cordially, %s team') % (self.env.user.company_id.name),
```

You can then remove the `get_default_message` method

Reply:
@ivantodorovich : I have Translated the default value but we can't remove reason is that we can't access the self in default param.